basics/fading/fading.toit

```diff
@@ -0,0 +1,33 @@
+/**
+Fading LED example code.
+
+This example increases and decreases the lighting level of a LED over and over again.
+
+How to wire the LED: 
+  - Place a 220ohm resistor in between the anode of the LED and GPIO19 and connect the cathode to ground.
+  - The resistor will increase the life of your LED.
+*/
+
+import gpio.pwm as gpio
+import gpio
+
+/// Defining the GPIO pin for the LED.
+LED ::= 19
+
+/// The configured range of the PWM.
+DUTY_MIN ::= 0.0
+DUTY_MAX ::= 1.0
+CHANGE_RATE ::= 0.01  // The amount of change every 5ms.
+
+main:
+  step := CHANGE_RATE
+  duty := DUTY_MIN
```

Comment:
I would move those two just before the while loop.
They are modified in the loop, so moving them closer there makes sense to me.
I would even have an empty line between the non-changing and the changing local.

Reply:
Fixed